apps/frontend/src/modules/points-of-interest/points-of-interest-api.ts

```diff
@@ -36,6 +36,23 @@ class PointOfInterestApi extends BaseHTTPApi {
 
 		return (await response.json()) as APIResponse<PointsOfInterestResponseDto>;
 	}
+
+	public async getById(
+		id: number,
+	): Promise<APIResponse<PointsOfInterestResponseDto>> {
+		const response = await this.load<APIResponse<PointsOfInterestResponseDto>>(
+			this.getFullEndpoint(
+				`${PointsOfInterestApiPath.ROOT}${id.toString()}`,
+				{},
+			),
```

Comment:
```suggestion
			this.getFullEndpoint(
				PointsOfInterestApiPath.ID,
				{id: id.toString()},
			),
```
And for the `PointsOfInterestApiPath`:
```
const PointsOfInterestApiPath = {
  ID: "/:id",
	ROOT: "/",
} as const;
```